production/helm/loki/values.yaml

```diff
@@ -18,7 +18,8 @@ annotations: {}
 tracing:
   jaegerAgentHost:
 
-config:
+# config: null
+configDefaults:
```

Comment:
I think I would prefer to have the file (string) as non default and keep config the default.

may be:

```
#configFile: |
# example of file config
config:
  auth_enabled: false
```

I would be nice to also add information in the readme about this changes, explaining the benefit and how to use it.